### PR TITLE
Update sample reports for v1.4.0 release

### LINKED ISSUE #1246: Generate updated sample reports for Iceberg
# 💡 Summary #

After the version bump, generate a new sample report from the latest ScubaGear version and replace the existing sample reports in the repository with them.

## Motivation and context ##

This is useful to ensure that the sample reporting reflects the latest results from the new release of the tool.

## Implementation notes ##

Please provide details for implementation, such as:

- Install the latest pre-release version of ScubaGear on a test machine
- Generate a report against a test tenant using user credentials
- Validate the report results
- Sanitize tenant and user details from the JSON, CSV, and HTML reports

## Acceptance criteria ##

How do we know when this work is done?

- [x] A new sample report reflecting the latest pre-release version is successfully added to the repository
- [x] All tenant and user details have been reviewed and removed or anonymized from the sample report

### PR DESCRIPTION
**Pull request by schrolla**
#  #
  

## 🗣 Description ##

Replaces the old sample reports with newly generated versions that incorporate changes and updates made up to v1.4.0.

## 💭 Motivation and context ##

Sample reports can be referenced by users to better understand what sort of output the tool can generate.

Closes #1246 

## 🧪 Testing ##

Sample reports were generated against a test tenant and an automated redaction script followed by manually editing the JSON to sanitize any user and tenant details.  Then, `Invoke-ScubaCached -Login:$false -ExportProvider:$false -M365Environment gcc -ProductNames * -MergeJson -OutPath <sample_outfolder>` is run which will use the existing out folder's JSON file as input and replace the other output files in that directory with those from the cached run.  The same command was run again without the `-MergeJson` option to generate the full range of possible output files possible depending on parameters.

Test the changes by checking out a local copy of the branch and manually reviewing the JSON, CSV, and HTML output files with local editor/browser.  Do note that the links to SCBs in the HTML report will generate a 404 Not Found until the actual release is published since the tag doesn't yet exist.  To check the links, just be sure that they are well-formed and include the v1.4.0 tag as the correct version.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] Changes are sized such that they do not touch excessive number of files.
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] These code changes follow the ScubaGear [content style guide](https://github.com/cisagov/ScubaGear/blob/main/CONTENTSTYLEGUIDE.md).
- [x] Related issues these changes resolve are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels added.
- [x] All relevant project fields are set.
- [x] All relevant repo and/or project documentation updated to reflect these changes.
- [x] Unit tests added/updated to cover PowerShell and Rego changes.
- [x] Functional tests added/updated to cover PowerShell and Rego changes.
- [x] All relevant functional tests passed.
- [x] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [x] PR passed smoke test check.
- [x] Feature branch has been rebased against changes from parent branch, as needed

  Use `Rebase branch` button below or use [this](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) reference to rebase from the command line.
- [x] Resolved all merge conflicts on branch
- [x] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [ ] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

File: PowerShell/ScubaGear/Sample-Reports/ProviderSettingsExport.json
[the remark this answers is not in the document]
Comment (dagarwal-mitre) [reply PowerShell/ScubaGear/Sample-Reports/ProviderSettingsExport.json:1]: Additionally, on line 1316 and line 2120 there is a mention of CISA Auth Strength. Considering everywhere else we have redacted CISA, is there a reason why these are still CISA not redacted?

**Review by dagarwal-mitre — CHANGES_REQUESTED** [review]
Everything looks good, 2 minor comments for the ProviderSettingsExport file.

**Review by dagarwal-mitre — APPROVED** [review]
Looks good!

**Review by mitchelbaker-cisa — APPROVED** [review]
Went through and didn't see anything major that stuck out. Approving but wanted to note a few areas I wasn't 100% on. 

ProviderSettingsExport.json:
- line 2958: cap_table_data is not redacted, granted there isn't necessarily sensitive data here but noting just in case
- line 17986: service principal uuid is displayed
- lines 735, 18002, 18021, 18035, 18065, 18464: tenantId is displayed

**Pull request merged**